test/verify/check-machines

```diff
@@ -370,6 +380,71 @@ class TestMachines(MachineCase):
             print("Libvirt version does not support disk statistics")
             b.wait_present("#vm-{0}-disksstats-unsupported".format(name))
 
+    def testLibvirt(self):
+        b = self.browser
+        m = self.machine
+
+        libvirtServiceName = self.getLibvirtName()
+
+        def checkLibvirtEnabled():
+            try:
+                m.execute("systemctl -q is-enabled {0}".format(libvirtServiceName))
+                return True
+            except:  # exception if return code != 0
+                return False
+
+        self.startVm("subVmTest1")
+        self.login_and_go("/machines")
+
+        b.wait_in_text("body", "Virtual Machines")
+        b.wait_present("tbody tr th")
+        b.wait_in_text("tbody tr th", "subVmTest1")
+
+        b.wait(lambda: checkLibvirtEnabled())
+        m.execute("systemctl disable {0}".format(libvirtServiceName))
+        print(self.getLibvirtName())
+        m.execute("systemctl stop {0}".format(libvirtServiceName))
+        b.wait(lambda: not checkLibvirtEnabled())
+        b.wait_present("#slate-header")
+        b.wait_in_text("#slate-header", "Virtualization Service (libvirt) is Not Active")
+        b.wait_present("#enable-libvirt:not(:checked)")
+        b.set_checked("#enable-libvirt", True)
+        b.click("#start-libvirt")
+
+        b.wait_in_text("body", "Virtual Machines")
+        b.wait(lambda: checkLibvirtEnabled())
+        b.wait_present("tbody tr th")
+        b.wait_in_text("tbody tr th", "subVmTest1")
+
+        m.execute("systemctl stop {0}".format(libvirtServiceName))
+        b.wait_present("#slate-header")
+        b.wait_in_text("#slate-header", "Virtualization Service (libvirt) is Not Active")
+        b.wait_present("#enable-libvirt:checked")
+        b.set_checked("#enable-libvirt", False)
+        b.click("#start-libvirt")
+        b.wait(lambda: not checkLibvirtEnabled())
```

Comment:
If you move this down one line (i. e. wait for the UI to update first), it should again not require waiting.